### PR TITLE
Add Unicorn Worker Settings.

### PR DESCRIPTION
**Pull request by olo-dw**
That can be useful to set the Unicorn Worker.

### CODE REVIEW

**Comment by olo-dw** [comment]
Rebased.

File: templates/gitlab.rb.j2 [hunk 1]
```diff
@@ -34,6 +34,17 @@ gitlab_rails['ldap_password'] = '{{ gitlab_ldap_password }}'
 gitlab_rails['ldap_allow_username_or_email_login'] = true
 gitlab_rails['ldap_base'] = '{{ gitlab_ldap_base }}'
 
+##################
+# GitLab Unicorn #
```
Comment (geerlingguy) [inline templates/gitlab.rb.j2]: For this section, can you just leave one comment line, like `# GitLab Unicorn settings` — that would better match the formatting of the rest of the file.

**Comment by madeinoz67** [comment]
@geerlingguy I would like to see your idea of gitlab_extra_settings and extending the template, as Ive wanted to make additional changes to gitlab settings, i.e. enable prometheus and changes to PGSQL memory settings through this role however they are currently not supported.  

I think your approach would allow for any future requirement changes without the need for specific implementations in the role, well at least it can be covered as a workaround until something is specifically added to the role.

**Comment by robbintt** [comment]
I would consider using the "assemble" ansible module for appending more
settings to this role. Then you can easily rebase your fork when this
project updates. You just need a second template file and an assemble task
that assembles the default template and yours.  You might need to modify
the template copy task, I haven't looked at the details. Any thoughts?

**Comment by madeinoz67** [comment]
thanks @robbintt, While I wasn't aware of this module (am now) I think should work nicely until something more official comes through from @geerlingguy  

I will have a play

**Comment by olo-dw** [comment]
#105 should be the right answer.

**Comment by olo-dw** [comment]
This merge request is too old. And I do think that the #105 is the real good proposition.

**Pull request closed without merging**